### ISSUE TITLE
List of Assets on Android App doesn't display correctly [Android, version 2.2.1]

### ISSUE THREAD

**Issue by laisee** [Power-Trade/api-docs#25]
List of Assets on Android App doesn't display correctly

When loading the Assets display on Android the list shows all assets but only has details for assets I own. 

Suggest the list of assets should be based on ones the users own. Possibly display all assets but keep users assets at top of the list.

![android-assets-display](https://github.com/Power-Trade/api-docs/assets/5905130/bdb7a2ed-a6d7-4b71-8a15-52a1a5dcc678)

**Comment by laisee** [comment]
**Update**

Sorting improved to display users assets with balances first plus a few others like AVAX, BTC. 

However the trailing set of assets looks incomplete - no symbols(coins) or values displayed so all entries look the same.

On IOS the "My Assets" display is limited to Spot market assets.
 
On Android now looks like this:

![android-assets-nada](https://github.com/Power-Trade/api-docs/assets/5905130/050908a1-a6bf-404c-937a-586ce90155ce)

**Comment by trungpwt** [comment]
Yes, you're on an old version, I will publish a new version that already includes bug fixes for that.
Thanks.
